resources/js/alternatingCase.js

```diff
@@ -0,0 +1,26 @@
+var toUpper = require('./toUpper'); // requiring custom toUpper method
+var toLower = require('./toLower'); // requiring custom toLower method
+
+// setting alternatingCase method to string protoype
+
+String.prototype.alternatingCase = function() {
+  var checkString = /\w/g; // setting regex pattern to match agains't string
```

Comment:
This is not a very good name for a variable. it is rather a good name for a function